app-localize-behavior.html

```diff
@@ -227,6 +232,12 @@
     };
   },
 
+  __languageChanged: function(newLang, oldLang) {
+    if (newLang === oldLang) return;
+    this.language = newLang;
+    this.fire('app-localize-behavior-language-changed', newLang);
```

Comment:
I could do it that way, but then the event name might be more likely to collide with an event named "language-changed" fired by some other component. Which approach do you prefer?